components/badge.scss

```diff
@@ -0,0 +1,7 @@
+$fd-icons-path: "../scss/icons/";
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/components/badge";
+@import "../scss/core/elements";
```

Comment:
Importing the entire core elements file is not what we want to do here. We need to find out what from this file is necessary to render links properly and bring those into the link class definitions -- either via common mixins or directly. The whole point of self-contained styles is not only bring along what is needed and not do the changes using elements.